Doc/library/typing.rst

```diff
@@ -125,7 +125,7 @@ Note that ``None`` as a type hint is a special case and is replaced by
 NewType
 =======
 
-Use the :class:`NewType` helper class to create distinct types::
+Use the :class:`NewType` helper to create distinct types::
```

Comment:
It's an implementation detail whether it is a class or a function, so avoid being specific.

Reply:
An alternative would be "helper callable", but I think "helper" is also fine.